### DISCUSSION TITLE
Kong vitals not working

### DISCUSSION THREAD

**Discussion by DipayanP007** [Kong/kong#8398] [Help] [answered]
Hey I would like to ask is Kong vitals entirely an enterprise feature or is it available without it? I am a bit confused about it. Though the doc says it is enterprise but the kong manager is provided with OSS. So can you please clarify if it is available to us?

**Comment by felderi** [comment, marked as answer]
Hi!  Thanks for using Kong.  

Vitals is a Kong Enterprise only feature.  With OSS, you can obtain transactional metrics from Kong using the OSS logging [plugins](https://docs.konghq.com/hub/#logging).  The advantage [Vitals](https://docs.konghq.com/gateway/2.7.x/vitals/) provides is reporting along with transactional metrics, Kong gateway health and performance metrics.

Kong Manager is provided with the [Freemium](https://docs.konghq.com/gateway/2.7.x/install-and-run/) version of Kong, not the OSS version of Kong.

Hope this helps.  Let us know if you have any other questions.

Fel.
